### ISSUE TITLE
Multi-line format does not work for customized emmet snippets.

### ISSUE THREAD

**Issue by robcca** [emmetio/emmet#655]
I am creating customized emmet snippets for html in a snippets.json file and the single-line ones work perfectly. This is in VS Code.

I'm trying to create a multi-line snippet in the same file and for every variation taken from a working regular snippets file (after adjusting it's syntax for emmet) the emmet parser gives an error message. There seems to be no mention of multi-line snippets in the emmet documentation.

Is this even possible?

**Comment by sergeche** [comment]
You should add newlines inside text nodes, e.g. `{foo\nbar}`

**Comment by robcca** [comment]
> You should add newlines inside text nodes, e.g. `{foo\nbar}`

Thank you, that will work for the question as I wrote it. 

I should have asked for a way to create an easily readable snippet. Since I couldn't find a way to do that with a customized emmet snippet I created a user snippet and added settings[^1] to settings.json to give priority to user snippets that have the same abbreviation as an emmet snippet.

For anyone who might be interested, I put the snippet file up as a gist, here:

https://gist.github.com/robcca/9170a327b5ad3d448159fdb7a9865982

[^1] The settings to put the user snippet at the top of the suggestions list, and selected, with the emmet suggestion second, are:
```json
"editor.snippetSuggestions": "top",
"emmet.showSuggestionsAsSnippets": false,
```
